src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualHub.md

```diff
@@ -0,0 +1,330 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermvirtualhub
+schema: 2.0.0
+---
+
+# New-AzureRmVirtualHub
+
+## SYNOPSIS
+Creates an Azure VirtualHub resource.
+
+## SYNTAX
+
+### ByVirtualWanObject (Default)
+```
+New-AzureRmVirtualHub -ResourceGroupName <String> -Name <String> -VirtualWan <PSVirtualWan>
+ -AddressPrefix <String> -Location <String>
+ [-HubVnetConnection <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSHubVirtualNetworkConnection]>]
```

Comment:
-HubNetConnection should be an array.

Reply:
Already fixed in code. Regenerated help.